common/sgx/verifier.c

```diff
@@ -641,7 +641,9 @@ static oe_result_t _verify_evidence(
     {
         report_body = evidence_buffer;
         report_body_size = sizeof(sgx_report_t);
-        custom_claims_buffer = report_body + report_body_size;
+        custom_claims_buffer = report_body_size < evidence_buffer_size
+                                   ? report_body + report_body_size
+                                   : NULL;
         custom_claims_buffer_size = evidence_buffer_size - report_body_size;
```

Comment:
If evidence_buffer_size < report_body_size, custom_claims_buffer_size should be 0. Same below.

Reply:
That's correct, but the converse is not true: when `evidence_buffer_size - report_body_size` is 0, then `custom_claims_buffer` is not `NULL` (but `report_body`).

Reply:
How about checking custom data availability first then setting `custom_claims_buffer_size` and `custom_claims_buffer`? That feels more straightforward to me. Same below.